test/nodes/test_prompt_node.py

```diff
@@ -89,7 +89,7 @@ def test_prompt_template_repr():
 
 @pytest.mark.unit
 def test_prompt_node_with_custom_invocation_layer_from_string():
-    model = PromptModel("fake_model", invocation_layer_class="test.nodes.test_prompt_node.CustomInvocationLayer")
+    model = PromptModel("fake_model", invocation_layer_class=CustomInvocationLayer)
```

Comment:
So in theory PromptModel only has to select "fake_model" here and this test would pass? If so, let's make it that way?

Reply:
Right! Let's try that